tests/MyGame/Example/Any.go

```diff
@@ -76,3 +76,22 @@ func (rcv Any) UnPack(table flatbuffers.Table) *AnyT {
 	}
 	return nil
 }
+
+func AnyVerify(verifier *flatbuffers.Verifier, typeId byte, tablePos flatbuffers.UOffsetT) bool {
+	var result bool
+
+	switch Any(typeId) {
+	case AnyMonster:
+		result = MonsterVerify(verifier, tablePos)
+		break
```

Comment:
In `Go`, these `break` statements are not needed

Reply:
fixed